### ISSUE TITLE
PGM chapter still using deprecated scripts folder

### ISSUE THREAD

**Issue by murphyk** [probml/pyprobml#914]
Fig 4.2 (made by https://colab.research.google.com/github/probml/pyprobml/blob/master/notebooks/book2/04/student_pgm.ipynb
) uses  the deprecated idiom
```
!wget -q https://raw.githubusercontent.com/probml/pyprobml/master/scripts/pyprobml_utils.py
!wget -q https://raw.githubusercontent.com/probml/pyprobml/master/scripts/pgmpy_utils.py
```

**Comment by patel-zeel** [comment]
Yes, we can automatically fix this via a script. Reassigning this to @karm-patel as he has done this before.

**Comment by karm-patel** [comment]
Fixed in #935
